tools/gulp/tasks/build-release.ts

```diff
@@ -1,6 +1,29 @@
 import {task} from 'gulp';
-import {composeRelease, sequenceTask} from 'lib-build-tools';
+import {mkdirpSync, writeFileSync} from 'fs-extra';
+import {buildConfig, composeRelease, sequenceTask} from 'lib-build-tools';
+import {join} from 'path';
+import {Bundler} from 'scss-bundle';
 import {flexLayoutPackage} from '../packages';
+
+const distDir = buildConfig.outputDir;
+const {sourceDir} = flexLayoutPackage;
+
+/** Path to the directory where all releases are created. */
+const releasesDir = join(distDir, 'releases');
+
+// Path to the release output of material.
+const releasePath = join(releasesDir, 'flex-layout');
+
+// Matches all SCSS files in the different packages. Note that this glob is not used to build
+// the bundle. It's used to identify Sass files that shouldn't be included multiple times.
+const allScssDedupeGlob = join(buildConfig.packagesDir, '**/*.scss');
+
+// The entry-point for the scss theming bundle.
+const themingEntryPointPath = join(sourceDir, 'core', 'sass', '_layout-bp.scss');
+
+// Output path for the scss theming bundle.
+const themingBundlePath = join(releasePath, '_mq.scss');
```

Comment:
I recently started disliking the `_` prefix to indicate private. 
Perhaps we should not use such conventions for our file names?

Reply:
Discussed offline that this is an unfortunate SASS convention